inst/sql/impact_method2_metadata/coverage_pop.sql

```diff
@@ -0,0 +1,33 @@
+SELECT tab1.*,
```

Comment:
Can you indent this to make it easier to read?

Reply:
Also consider dropping the .* unless you *always* really want all columns (if so, that's fine).

Reply:
Comment to indicate that `tab1` is a pretend table made in this query (though consider a better name)

Reply:
Done.